src/plus/integrations/authentication/cloudIntegrationService.ts

```diff
@@ -35,17 +35,26 @@ export class CloudIntegrationService {
 
 	async getConnectionSession(
 		id: IntegrationId,
-		refresh: boolean = false,
+		deprecatedAccessTokenToRefresh?: string,
 	): Promise<CloudIntegrationAuthenticationSession | undefined> {
+		const refresh = Boolean(deprecatedAccessTokenToRefresh);
 		const cloudIntegrationType = toCloudIntegrationType[id];
 		if (cloudIntegrationType == null) {
 			Logger.error(`Unsupported cloud integration type: ${id}`);
 			return undefined;
 		}
+		const reqInitOptions = refresh
```

Comment:
```suggestion
		const cloudIntegrationType = toCloudIntegrationType[id];
		if (cloudIntegrationType == null) {
			Logger.error(`Unsupported cloud integration type: ${id}`);
			return undefined;
		}
		const reqInitOptions = refreshToken != null
```

Reply:
No need to define a variable since we're only using the check once - can just inline the check on `reqInitOptions`

Reply:
It's not once. It's used here:
![image](https://github.com/gitkraken/vscode-gitlens/assets/83944326/7e7e5d0f-ab1a-4372-b90c-3d5f3d178217)

Reply:
Ah I see, thanks. Let's keep it then.